vue/configurator/src/components/Configurator.vue

```diff
@@ -68,6 +68,72 @@
               v-model="prop.value"
               size="large"
             />
+            <!-- data-media-background has a type of JSON string -->
+            <el-row
+              v-else-if="prop.type === WidgetPropType.INTERFACE"
+              style="
+                align-items: center;
+                border-color: lightgrey;
+                border-style: solid;
+                border-width: thin;
+                border-radius: 5px;
+                height: 150px;
+                width: 250px;
```

Comment:
minor nit: can you add these styles to the scoped styles tag of this SFC ?

Reply:
Done! Added it in the scoped style by giving it a class called `prop-groupings` in case theres any future prop that has many variables to visually group together